account_chart_update/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# Copyright 2016 Jacques-Etienne Baudoux <je@bcim.be>
+# Copyright 2016 Sylvain Van Hoof <sylvain@okia.be>
+# Copyright 2015-2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': "Detect changes and update the Account Chart from a template",
+    "summary": "Wizard to update a company's account chart from a template",
```

Comment:
Nothing that is blocking but by aesthetic would be better to use or simple quote or double quote but not mix